src/database/queries/reviews.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+const reviews = require('../models/index').Review
+
+exports.findAll = async (filter, attributes = {}) => {
+	try {
+		const res = await reviews.findAll({
+			where: filter,
+			attributes,
+			raw: true,
+		})
+
+		return res
+	} catch (error) {
+		return error
+	}
+}
+exports.countDistinct = async (filter, attributes = {}) => {
```

Comment:
Return the unique count of resource id. Also check if we can group and count distinct